docs/src/visualization.md

```diff
@@ -16,11 +16,11 @@ Follow these steps to view the exported VTK files in ParaView:
 5. Hold the left mouse button to move the solution around.
 
 You will now see the following:
-![image](https://github.com/svchb/TrixiParticles.jl/assets/10238714/45c90fd2-984b-4eee-b130-e691cefb33ab)
+![image](https://github.com/user-attachments/assets/108a55c5-888c-457c-8308-eb4a533d0833)
```

Comment:
I find this confusing. There is no axis grid and particles are all the same color when first opening ParaView.

Reply:
These pictures are not new. They have just not been accessible since we switched from a private repo to a public one.

Reply:
Yes we did but it would be better if you would say then when we are at it lets replace it with these better images.
Note: I cannot change it at all since it is themed through the OS.